src/core/search/search.cc

```diff
@@ -488,6 +488,9 @@ void FieldIndices::CreateIndices(PMR_NS::memory_resource* mr) {
           vector_index = make_unique<FlatVectorIndex>(vparams, mr);
 
         indices_[field_ident] = std::move(vector_index);
+        LOG_FIRST_N(WARNING, 1)
+            << "Created vector index '" << field_ident
+            << "', but the computed score field '__vector_score' is not supported yet";
```

Comment:
so I think it's a confusing warning. 
(1) you can sort by alias
(2) results are sorted by default, so only sortby desc makes sense